revocation/ocsp/ocsp.go

```diff
@@ -177,7 +177,7 @@ func executeOCSPCheck(cert, issuer *x509.Certificate, server string, opts Option
 		reader := bytes.NewReader(ocspRequest)
 		resp, err = opts.HTTPClient.Post(server, "application/ocsp-request", reader)
 	} else {
-		encodedReq := base64.URLEncoding.EncodeToString(ocspRequest)
+		encodedReq := url.QueryEscape(base64.StdEncoding.EncodeToString(ocspRequest))
```

Comment:
Do you want to also change line 176?

Reply:
You may also take a look at [url.PathEscape](https://pkg.go.dev/net/url#PathEscape).